ui/components/BigNumber/index.js

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import { Grid, Box, Typography, Button, CardContent, CardActions, Card } from "@mui/material/";
+
+import PropTypes from "prop-types";

```

Comment:
It is really good of you to use prop-types @Meghana-12 , but we have not yet settled on a solution for types. Our intention was to wait for sometime and then, in the future, move to typescript completely. So, I think its better not to waste time incorporating a different style when it is obvious that we have to ditch it.

Reply:
Oh okay! I was getting eslint errors so I added it. Will remove it then!

Reply:
I see. We have to change the eslint config then.